change/@ni-eslint-config-angular-ebf2bbaa-353f-4c86-81c2-7956d462fc41.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Did you sanity check the changes against an existing project (ie with the packages tgz) that is still on 14 to make sure nothing funny happens?

Reply:
sounds liek that was addressed here 👍 https://github.com/ni/javascript-styleguide/pull/129#discussion_r1380445691